src/cycax/cycad/Nut_Cut_Out.py

```diff
@@ -0,0 +1,36 @@
+from Location import Location
+
+
+class Nut_Cut_Out(Location):
+    """Class for holding the data for nuts."""
+
+    nuts = {
+        "type": 3,
+        "Diameter": 6.01,
+        "Height": 2.4,
+        "type": 6,
+        "Diameter": 11.05,
+        "Height": 5.2,
+    }
+
+    def __init__(
+        self, side: str, x: float, y: float, z: float, type: float, depth: float
+    ):
+        """THis method will create a nut."""
+        Location.__init__(self, x, y, z, side)
+        self.type = type
+        self.depth = depth
+
+    def export(self):
+        """This method will create a dictionary of the nut."""
+        dict_nut = {
+            "name": "nut",
+            "type": "cut",
+            "side": self.side,
+            "x": str(self.x),
```

Comment:
Would float not be better than str for x,y, and z.?

Reply:
json has trouble handling numbers that are not integers. It could not process the super accurate 64bit floats being produced by the external part stl calculation. If I put it as a string it stopped complaining